### PR TITLE
Add desired tags where we want them to VPC and Subnets

### PR DESCRIPTION
**Pull request by pturley**
Review and merge https://github.com/tablexi/terraform_modules/pull/100 before this.

### CODE REVIEW

**Comment by phoolish** [comment]
I'd like to have #100 merged first before reviewing

**Comment by pturley** [comment]
@phoolish ready for another look

File: aws/vpc/subnets/main.tf [hunk 1]
```diff
@@ -26,35 +26,30 @@ data "template_file" "az_to_netnum" {
...
-  vpc_id            = "${var.vpc_id}"
-  cidr_block        = "${cidrsubnet(data.aws_vpc.current.cidr_block, var.newbits, var.netnum_offset + element(data.template_file.az_to_netnum.*.rendered, count.index))}"
-  availability_zone = "${element(data.aws_availability_zones.available.names, count.index)}"
-  count             = "${length(data.aws_availability_zones.available.names)}"
-
-  tags {
-    Name = "${var.name}-${element(data.aws_availability_zones.available.names, count.index)}"
-  }
-
+  count                   = "${length(data.aws_availability_zones.available.names)}"
```
Comment (phoolish) [inline aws/vpc/subnets/main.tf:42]: Why is this still showing up then?

Comment (phoolish) [reply aws/vpc/subnets/main.tf:42]: Never mind reordering

**Review by phoolish — APPROVED** [review]
🎉

**Pull request merged**